frontend/js/marc-record-merge-postmerge-service.js

```diff
@@ -447,6 +462,29 @@ export function removeExtra035aFromMerged(preferredRecord, otherRecord, mergedRe
   };
 }
 
+export function add080VersionCode (preferredRecord, otherRecord, mergedRecordParam) {
+  const mergedRecord = new MarcRecord(mergedRecordParam);
+
+  mergedRecord.fields.filter(field => field.tag === '080')
+    .forEach(field => {
+      const hasCode2 = field.subfields.findIndex(sub => sub.code === '2');
+      const hasCode9FKeep = field.subfields.findIndex(sub => sub.code === '9' && sub.value === 'FENNI<KEEP>');
```

Comment:
No need to check for existence of subfield 9 (This wan't in the spec?)

Reply:
I tested it and found some records that have multiple 080 fields and would be weird to add Fenni stuff if it was not Fenni field?